src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -35,6 +35,8 @@ DnsStreamHandler::DnsStreamHandler(const std::string &name, InputStream *stream,
     if (!_pcap_stream && !_mock_stream && !_dnstap_stream) {
         throw StreamHandlerException(fmt::format("DnsStreamHandler: unsupported input stream {}", stream->name()));
     }
+
+    _g_enabled = _process_dns_groups(window_config);
```

Comment:
I tried to use `_g_enabled `inside `_process_dns_groups` so I could initialize it there. however, when `_process_dns_groups ` returns, `_g_enabled` is reinitialized as default, due to c++ initializer order

Reply:
So, I need to recall `_process_dns_groups` to initialize `_g_enabled` properly